common/src/main/java/net/draycia/carbon/common/channels/CarbonChannelRegistry.java

```diff
@@ -239,6 +249,16 @@ private void loadConfigChannels_(final CarbonMessages messages) {
                 this.logger.info("Default channel is [" + channelKey + "]");
             }
 
+            if (this.configChannels.contains(channelKey)) {
+                this.logger.warn("Channel with key [" + channelConfigFile + "] has already been registered, skipping");
+
+                if ("partychat".equals(channelKey.value())) {
```

Comment:
I think we can just check this in the mcmmo integration eligible or register method, and throw an exception

Reply:
also we should throw an exception in registerSpecialHandler on duplicate key to avoid mistakes when adding handlers